.github/workflows/build.yaml

```diff
@@ -100,7 +99,7 @@ jobs:
           go-version: ${{ env.go_version }}
 
       - name: Setup go mockgen
-        run: go install github.com/golang/mock/mockgen@v1.6
+        run: go install go.uber.org/mock/mockgen@v0.4.0
 
```

Comment:
Why do we need `mockgen` for a release?  (Does `goreleaser` run tests?)

Reply:
I believe it is required for the [go generate ./...](https://github.com/arcalot/arcaflow-container-toolkit/blob/60d095cd5b35ff3bbd56dd95bcda0524ca900d8c/.goreleaser.yaml#L6) in the .goreleaser file, but maybe it could be removed.